website/source/docs/providers/azurerm/r/virtual_machine_scale_sets.html.markdown

```diff
@@ -106,6 +106,122 @@ resource "azurerm_virtual_machine_scale_set" "test" {
 }
 ```
 
+## Example Usage with Managed Disks
+
+```
+resource "azurerm_resource_group" "test" {
+  name     = "acctestrg"
+  location = "West US 2"
+}
+
+resource "azurerm_virtual_network" "test" {
+  name                = "acctvn"
+  address_space       = ["10.0.0.0/16"]
+  location            = "West US 2"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+
+resource "azurerm_subnet" "test" {
+  name                 = "acctsub"
+  resource_group_name  = "${azurerm_resource_group.test.name}"
+  virtual_network_name = "${azurerm_virtual_network.test.name}"
+  address_prefix       = "10.0.2.0/24"
+}
+
+resource "azurerm_public_ip" "demoterraformips" {
```

Comment:
The resource names are inconsistent, and this example gives errors - probably best to change all resource names to "test" like I've seen with most examples

* resource 'azurerm_public_ip.demoterraformips' config: unknown resource 'azurerm_resource_group.demoterraform' referenced in variable azurerm_resource_group.demoterraform.name
* resource 'azurerm_lb.demoterraformlb' config: unknown resource 'azurerm_resource_group.demoterraform' referenced in variable azurerm_resource_group.demoterraform.name
* resource 'azurerm_lb_nat_pool.lbnatpool' config: unknown resource 'azurerm_resource_group.demoterraform' referenced in variable azurerm_resource_group.demoterraform.name
* resource 'azurerm_virtual_machine_scale_set.test' config: unknown resource 'azurerm_subnet.demoterraformsubnet' referenced in variable azurerm_subnet.demoterraformsubnet.id
* resource 'azurerm_lb_backend_address_pool.bpepool' config: unknown resource 'azurerm_resource_group.demoterraform' referenced in variable azurerm_resource_group.demoterraform.name